API/Hardware_API.md

```diff
@@ -68,6 +68,61 @@ If you're newer to hardware and these functions look like alphabet soup to you,
 
 By default, all of the pins are pulled high if not specifically set.
 
+### Pullup and Pulldown pins
+
+Pins 2-7 on both Ports are available for pullup and pulldown.
+
+The basic function of a pull-up resistor is to insure that given no other input, a circuit assumes a default value. The 'pullup' mode pulls the line high and the 'pulldown' mode pulls it low.
+
+### Usage
+```js
+// Invoke this method to set the pull mode. pullType can be - pullup, pulldown or none.
+pin.pull(pullType, callback);
+```
+
+Example of the `pin.pull(mode)` command using a pushbutton. The code example given below turns on the Blue LED of the Tessel module when the pushbutton is pressed and turns off the Blue LED when the pushbutton is released.
```

Comment:
This should just say `Example:` to stay consistent with the other Usage documentation. Then link to Tutorial page after the example, i.e.:

```markdown
[Learn more about using the Pullup / Pulldown API.](/Tutorials/pinpull.html)
```